test/integration/image-future/default/test/index.test.js

```diff
@@ -992,6 +992,24 @@ function runTests(mode) {
         )
       ).toBe('100%')
     })
+    it('should add position styles to fill images', async () => {
+      expect(
+        await browser.eval(`document.getElementById("fill-image-1").style.left`)
+      ).toBe('0px')
+      expect(
+        await browser.eval(`document.getElementById("fill-image-1").style.top`)
+      ).toBe('0px')
+      expect(
+        await browser.eval(
+          `document.getElementById("fill-image-1").style.right`
+        )
+      ).toBe('0px')
+      expect(
+        await browser.eval(
+          `document.getElementById("fill-image-1").style.bottom`
+        )
+      ).toBe('0px')
```

Comment:
Lets change this to get the whole style so we will fail if more that the expected styles are added

```
await browser.elementById('fill-image-1').getAttribute('style')
```